docs/guides/tms.rst

```diff
@@ -0,0 +1,242 @@
+TMS Servers
+===========
+
+GeoPySpark is meant to work with geospatial data.  The most natural way to
+interact with these data is to display them on a map.  In order to allow for
+this interactive visualization, we provide a means to create Tile Map Service
+(TMS) servers directly from both GeoPySpark RDDs and tile catalogs.  A TMS
+server may be viewed using a web-based tool such as geojson.io_ or interacted
+with using the GeoNotebook_ Jupyter kernel. [#]_
+
+Note that the following examples rely on this common boilerplate code:
+
+.. code-block:: python
+
+   from geopyspark import geopyspark_conf
+   from geopyspark.geotrellis.tms import TMS
+   from pyspark import SparkContext
+
+   conf = geopyspark_conf(appName="demo")
+   sc = SparkContext(conf=conf)
+
+Basic Example
+-------------
+
+The most straightforward use case of the TMS server is to display a singleband
+layer with some custom color map.  This is accomplished easily:
+
+.. code-block:: python
+
+   from geopyspark.geotrellis.color import ColorMap
```

Comment:
Similar to the above comment - all import are generally done through the top, boilerplate code.